source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,493 @@
+   Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /usr/lib/systemd/system/nginx.service.
+   mkdir: created directory '/etc/uwsgi'
+   mkdir: created directory '/etc/uwsgi/vassals'
+   Created symlink /etc/systemd/system/multi-user.target.wants/uwsgi.service → /etc/systemd/system/uwsgi.service.
+   ALTER ROLE
+   sed: can't read /tmp/telemetrics-backend/collector/config.py: No such file or directory
+   cp: cannot stat '/tmp/telemetrics-backend/collector/config.py': No such file or directory
+   sed: can't read /tmp/telemetrics-backend/telemetryui/config.py: No such file or directory
+   cp: cannot stat '/tmp/telemetrics-backend/telemetryui/config.py': No such file or directory
+   Already using interpreter /usr/bin/python3
+   Using base prefix '/usr'
+   New python executable in /var/www/telemetry/venv/bin/python3
+   Not overwriting existing python script /var/www/telemetry/venv/bin/python (you must use /var/www/telemetry/venv/bin/python3)
+   Installing setuptools, pip, wheel...done.
+   INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+   INFO  [alembic.runtime.migration] Will assume transactional DDL.
+   INFO  [alembic.runtime.migration] Running upgrade  -> 3230c615d6e0, empty message
+   INFO  [alembic.runtime.migration] Running upgrade 3230c615d6e0 -> 466cf2f35d67, empty message
+
+   Install complete (installation folder: /var/www/telemetry)
+
+Once the installation is complete you can use your web browser and view the
+new server by opening the web browser on the system you installed the backend
+server onto and type in ``localhost`` in the address bar.  You should see a
+web page similar to the one shown in figure 1:
+
+.. figure:: figures/telemetry-backend-1.png
+   :scale: 50 %
+   :alt: Telemetry UI
+
+   Figure 1: :guilabel:`Telemetry UI`
+
+
+Redirect telemetry records
+==========================
+
+Telemetry records from your system are sent to the server location defined in
+the :file:`/usr/share/defaults/telemetrics/telemetrics.conf` configuration
+file. You can customize this by copying this file to
+:file:`/etc/telemetrics/telemetrics.conf` and changing the ``server=``
+setting to your new server location.
+
+#. Create the :file:`/etc/telemetrics` directory and change to it
+
+   .. code-block:: console
+
+      sudo mkdir -p /etc/telemetrics
+      cd /etc/telemetrics
+
+
+#. Copy the default :file:`telemetrics.conf` file to the new
+   :file:`/etc/telemetrics` directory
+
+   .. code-block:: console
+
+      sudo cp /usr/share/defaults/telemetrics/telemetrics.conf .
+
+#. Edit the new :file:`/etc/telemetrics/telemetrics.conf` file with your
+   editor using the :command:`sudo` directive and change the
+   :guilabel:`server=` setting to ``http://localhost/v2/collector`` and save
+   this change in the new file
+
+   .. code-block:: console
+
+      server=http://localhost/v2/collector
+
+   You can also use the fully qualified domain name for your server instead of
+   localhost
```

Comment:
`local host`.

Reply:
added the guilabel but localhost needs to remain one word since it is a valid address to use